src/components/date-picker/date-picker--single-calendar.html

```diff
@@ -0,0 +1,14 @@
+<!-- Basic with calendar -->
+<div class="bx--form-item">
+  <div data-date-picker="single" class="bx--date-picker">
+    <div class="bx--date-picker__container">
+      <label for="date-picker-1" class="bx--label">Label</label>
```

Comment:
Even though this is for demo, make sure all associated `for` and `id` attrs are unique